include/tvm/tir/usmp/utils.h

```diff
@@ -59,22 +60,26 @@ struct PoolInfoNode : public Object {
   Integer size_hint_bytes;
   /*! \brief The accessibility from each Target*/
   Map<Target, String> target_access;  // 'rw' or 'ro'
+  /*! \brief Whether pool is internally generated*/
+  Bool is_internal = Bool(false);
```

Comment:
out of curiosity any reason to use the TIR Bool node over just bool? FFI is all perfectly happy with plain old bool.

Reply:
nit: one extra sentence 'Internally generated pools are needed for ... and treated differently by ...' would help.

Reply:
Done